plugin/trino-hive/src/main/java/io/trino/plugin/hive/acid/AcidOperation.java

```diff
@@ -27,7 +27,8 @@
     CREATE_TABLE,
     DELETE,
     INSERT,
-    UPDATE;
+    UPDATE,
+    ANALYZE;
```

Comment:
```suggestion
    ANALYZE,
    /**/;
```

so that you don't need to "change" the last entry when adding a new one

Reply:
Changed.